flash/vision/classification/data.py

```diff
@@ -564,5 +522,6 @@ def from_filepaths(
             batch_size=batch_size,
             num_workers=num_workers,
             preprocess=preprocess,
+            seed=seed,
             **kwargs
```

Comment:
I had a bug because of that since kwargs are not check anywhere. One suggestion is to avoid kwargs and define explicit arguments (which helps readability and tracing bugs) or add checking in the constructor.

Reply:
we need kwargs here since users can subclass this and we still want all custom arguments to be passed through

Reply:
Yes, let's add this in another PR.